packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -143,7 +143,8 @@
     </template>
     <div
       v-if="docsMenuVariant === 'main'"
-      class="flex p-16px gap-24px"
+      data-cy="docs-menu-container"
+      class="flex p-16px gap-24px docs-menu-flex-direction"
```

Comment:
@sainthkh we've been using Tailwind CSS in the Vue components. According to their docs on [flex direction](https://tailwindcss.com/docs/flex-direction) and [responsive design](https://tailwindcss.com/docs/responsive-design), this should do the trick. Can you try it out? Thanks!
```suggestion
      class="flex p-16px gap-24px flex-col md:flex-row"
```

Reply:
AHHHH nevermind, I just re-read your GitHub description with the explanation on it not working

Reply:
@sainthkh I just tried it out with the Tailwind CSS did work for me, can you confirm? Make sure to refresh the launchpad to see changes since it doesn't hot reload

Reply:
Alrighty I put in the Tailwind CSS change and reorganized the tests to be scoped in a `responsive design` context block. I also broke them out into two tests to take advantage of the `percySnapshot` command in the `afterEach`. Sorry it took so long to get in and big thanks for your contribution 🙏